### PR TITLE
Add September and October 2023

### PR DESCRIPTION
**Pull request by Bestem0r**
Added monthly post about what we've done in webkom during September and October 2023.

### CODE REVIEW

File: _posts/2023-10-02-september-october.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+title: September og Oktober 2023
+author: Bestem0r
+---
+
+Hei og hogg! Ny månad, ny utviklarblogg! 🎉 Dei siste to månadane har me i Webkom jobba jamnt og trutt for å forbedre og vidareutvikle abakus.no og dei andre tenestene våre. 
+Som ny bloggansvarleg er det derfor med stor ære og glede at eg no kan presentere nettopp kva me i Webkom har jobba med i desse haustmånadane.
+
+Men før me går i gong, har du kanskje merka at noko er litt utanom det vanlege? Noko som skurrar litt når du les desse orda? Det stemmer! Dette og kommande innlegg kjem nemleg til å bli skrivne på det vakre landsmålet vårt, nynorsk! 🇳🇴
+
+Det har seg nemleg slik at etter ei rask undersøking, har eg fått inntrykk av at det er forsvinnande lite nynorsk nytta på abakus sine sider. Så får å bidra til å rette på dette har eg bestemt meg for å skrive utviklarbloggen på nynorsk! 🤓 (Skriv gjerne ein kommentar om du er ueinig i dette valget, så kan me ta ein diskusjon om det!)
```
Comment (ollfkaih) [inline _posts/2023-10-02-september-october.md:12]:
```suggestion
Det har seg nemlig slik at etter en rask undersøkelse, har jeg fått inntrykk av at det er forsvinnende lite nynorsk på abakus sine sider. Så for å bidra til å rette på dette har jeg bestemt meg for å skrive utviklarbloggen på norsk! 🤓 (Skriv gjerne en kommentar om du er uenig i dette valget, så kan vi ta en diskusjon om det!)
```

File: _posts/2023-10-02-september-october.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+### Lagt til felt for arrangement-organisator
+
+Arrangementansvarlege kan no legge til ein organisator for arrangementet sitt! 🎊
+
+![organisator](/images/posts/2023-10-02-organisator.png)
+
+### Lagt til sortering på adminsida til admissions
+
+Admissions har no fått sortering på adminsida si! 🤩
```
Comment (ollfkaih) [inline _posts/2023-10-02-september-october.md:59]:
```suggestion
Opptak har no fått sortering på adminsida si! 🤩
```

File: _posts/2023-10-02-september-october.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+| ![seksjonar](/images/posts/2023-10-02-statistikk-1.png) | ![grid](/images/posts/2023-10-02-statistikk-2.png) |
+
+
+### Lagt til felt for arrangement-organisator
+
+Arrangementansvarlege kan no legge til ein organisator for arrangementet sitt! 🎊
+
+![organisator](/images/posts/2023-10-02-organisator.png)
+
+### Lagt til sortering på adminsida til admissions
```
Comment (ollfkaih) [inline _posts/2023-10-02-september-october.md:57]:
```suggestion
### Lagt til sortering på adminsida til opptak
```

**Review by ollfkaih — APPROVED** [review]
I couldn't understand anything of what was written so I'm just approving it.

**Review by eikhr — APPROVED** [review]
Looks great!

File: _posts/2023-10-02-september-october.md (hunk 1 above)
Comment (ollfkaih) [reply _posts/2023-10-02-september-october.md:12]: Får > for

Comment (Bestem0r) [reply _posts/2023-10-02-september-october.md]: Done! @ollfkaih

**Review by haukkagu — APPROVED** [review]
LGTM. Remember to squash before merging!!!

**Review by ollfkaih — APPROVED** [review]

**Pull request merged**